.github/workflows/tests_wf.yml

```diff
@@ -27,16 +27,15 @@ env:
 jobs:
   tests:
     runs-on: ${{ inputs.os }}-latest
+    defaults:
+      run:
+        shell: bash
     steps:
     - name: Install BLAS on PyPy3.10
       if: startsWith(inputs.python-version, 'pypy')
       run: sudo apt install -y libopenblas-dev
-    - name: Set env variables for ubuntu
-      if: inputs.upload-coverage && startsWith(inputs.os, 'ubuntu')
+    - name: Set env variables
       run: echo "PYTEST_ARGS=--cov-append --cov=pydicom" >> "$GITHUB_ENV"
```

Comment:
Ah, this also avoids having to handle the env vars separately - I should use that elsewhere, too...